tests/conftest.py

```diff
@@ -179,6 +179,8 @@ def app_config(app_config, mock_datacite_client):
     for config_key in supported_configurations:
         app_config[config_key] = getattr(config, config_key, None)
 
+    app_config["THEME_SITENAME"] = "Invenio"
```

Comment:
Nice, even asserting that the sitename is properly used in the error message. :+1: